src/blobstreamx.cairo

```diff
@@ -147,9 +148,9 @@ mod blobstreamx {
             // load the tuple root at the given index from storage.
             let root: u256 = self.state_data_commitments.read(proof_nonce);
 
-            let mut data_root_bytes = BytesTrait::new_empty();
-            data_root_bytes.append_felt252(data_root.height);
-            data_root_bytes.append_u256(data_root.data_root);
+            let mut data_root_bytes = BytesTrait::new_empty()
```

Comment:
I don't think these variables need to be `mut` anymore with this setup. Hope that's not too much to change.

Reply:
Indeed they don't!